cli/cliutils/cliutils.go

```diff
@@ -1448,6 +1449,27 @@ func InvokeRestApi(httpClient *http.Client, method string, urlPath string, crede
 	}
 }
 
+// This function will return response directly, the resp need to be closed by caller
+// Set closeRequest to true if it is last chunk
+// Range header will be added (Range:bytes={startOffset}-{endOffset})
+func MMSGetChunkDataResponse(httpClient *http.Client, credentials string, objOrg string, objType string, objID string, startOffset int64, endOffset int64, closeRequest bool) *http.Response {
+	service := "Model Management Service"
+
+	dataUrlPath := path.Join("api/v1/objects/", objOrg, objType, objID, "/data")
+	url := GetMMSUrl() + "/" + dataUrlPath
+	apiMsg := http.MethodGet + " " + url
+
+	Verbose(apiMsg)
+
+	//httpClient := GetHTTPClient(config.HTTPRequestTimeoutS)
+
+	headers := make(map[string]string)
+	headers["Range"] = fmt.Sprintf("bytes=%d-%d", startOffset, endOffset)
+
+	resp := InvokeRestApi(httpClient, http.MethodGet, url, credentials, nil, service, apiMsg, headers, closeRequest)
```

Comment:
this function should be used for CLI only because it exit out of the program when error occurs.  needs a separate ` MMSGetChunkDataResponse` function for the agent that calls exchange.InvokeExchange() or exchange.InvokeExchangeRetryOnTransportError().